pkgs/development/libraries/fmt/default.nix

```diff
@@ -9,6 +9,12 @@ stdenv.mkDerivation rec {
     rev = "${version}";
     sha256 = "1cd8yq8va457iir1hlf17ksx11fx2hlb8i4jml8gj1875pizm0pk";
   };
+  patches = [
+    pkgs.fetchpatch {
```

Comment:
You need parentheses here, otherwise it is interpreted as being two elements of an array instead of a call.